samples/app-cache-meetings/nodejs/src/components/app-cache-tab.tsx

```diff
@@ -0,0 +1,126 @@
+// <copyright file="app-cache-tab.tsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useState } from "react";
+import React from "react";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+/// </summary>
+/// function logitem
+/// </summary>
+function logItem(action: string, actionColor: string, message: string) {
+  const newItem =
```

Comment:
not sure if other samples use fluent UI. feel free to use the same UI components instead of raw html.  Leave as is if all samples dont use the same UI widgets